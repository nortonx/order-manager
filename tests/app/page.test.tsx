import { render, screen } from '@testing-library/react';
import HomePage from '@/app/page';
import { describe, it, expect } from '@jest/globals';

describe('HomePage', () => {
  it('renders the home page', async () => {
    const { container } = render(<HomePage />);
    const homePage = screen.getByTestId('home-page');
    expect(homePage).toBeDefined();
    expect(container).toMatchSnapshot();
  });
});
