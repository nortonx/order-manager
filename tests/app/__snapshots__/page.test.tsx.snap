// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders the home page 1`] = `
<div>
  <div
    class=""
    data-testid="home-page"
  >
    <h1>
      Home Page
    </h1>
  </div>
</div>
`;
